backend/tracim_backend/config.py

```diff
@@ -784,8 +784,9 @@ def _load_email_config(self) -> None:
         )
 
         # Summary
-        self.EMAIL__SUMMARY__TEMPLATE__HTML = self.get_raw_config(
-            "email.summary.template.html", "{}/{}".format(template_dir, "summary_body_html.mak"),
+        self.EMAIL__NOTIFICATION__SUMMARY__TEMPLATE__HTML = self.get_raw_config(
+            "email.notification.summary.template.html",
+            "{}/{}".format(template_dir, "summary_body_html.mak"),
```

Comment:
Qu'est-ce qui a été fait à l'égard des anciennes configs sur le fait de changer le nom et le type de ce paramètre ? Il doit y avoir des exemples avec d'autres configs (ou au moins dans l'historique). L'idée, c'est que les anciens paramètres fonctionnent toujours, mais déclenchent un message de dépréciation et que les nouveaux paramètres ai des valeurs par défaut qui coïncide avec les valeurs par défaut des anciens paramètres.

Pour simplifier : image une instance de Tracim qui a un fichier de config **avant** cette modif : ce fichier de config va t-il permettre à Tracim de fonctionner dans cette nouvelle version ou est-ce que ça va poser un problème (au démarrage, ou à l'utilisation de ces nouveaux paramètres).

Si ce n'est pas possible d'être rétrocompatible, il faut signaler un breaking change et s'organiser avec l'équipe d'admin sys.

Reply:
Le paramètre n'est pas rétro-compatible, on pourrait le rendre compatible moyennant de la charge de travail. Cependant ce développement est une _suite_ de `feat/6106__daily_mail_notification` qui contient donc ce breaking change. Le breaking change est ici https://github.com/tracim/tracim/issues/6106#issuecomment-1450127660 et sera dans la documentation de migration.

The parameter isn't backward compatible, we could make it compatible with quite some work.
However this development is _following_ `feat/6106__daily_mail_notification` which contains the breaking change. The breaking change is here https://github.com/tracim/tracim/issues/6106#issuecomment-1450127660  and will be in the migration documentation.